.github/workflows/release.yml

```diff
@@ -76,9 +37,9 @@ jobs:
         with:
           tag: ${{ github.ref }}
           name: Release ${{ github.ref }}
-          body: ${{ steps.changelog.outputs.path }}
+          body: ${{ steps.build.changelog_outputs_path }}
           draft: true
-          artifacts: ./${{ steps.metadata.outputs.archive }}
+          artifacts: ./${{ steps.build.metadata_outputs_archive }}
           artifactContentType: application/zip
 
       - name: Add checksum to release
```

Comment:
We can put that into the step above.

Reply:
but then they have both `artifactContentType: raw`. we will see if this is a problem